semver/increment.ts

```diff
@@ -186,7 +186,7 @@ export function increment(
       // If it is a pre-release it will bump up to the same patch version.
       // 1.2.0-5 patches to 1.2.0
       // 1.2.0 patches to 1.2.1
-      if (version.prerelease.length === 0) {
+      if (version.prerelease?.length === 0) {
```

Comment:
Ditto. This seems introducing a bug. Please add test case to exercise this pattern with prerelease = `undefined`.

Reply:
done